### ISSUE TITLE
Bring back the quit function

### ISSUE THREAD

**Issue by voidzero** [ubergeek42/weechat-android#301]
Using 0.12-git, I noticed that you have removed the quit function. I know that you can disconnect and close and so forth, but I prefer the quickness of being able to press quit. Can you put it back, please?

**Comment by mhoran** [comment]
The quit button was removed as explained in #252, so I'm going to close this out as a duplicate.

However, to answer your question further: we removed the quit button because it is an Android anti-pattern. Other apps, e.g. Hangouts, don't have a quit option. Since we have persistent connections, we do have a Disconnect option. As @oakkitten describes in #252, the Quit button didn't actually do anything other than disconnect and pop the app off the app stack for some time. So if you want to "quit", just hit disconnect. The service will stop running but Android will manage the memory appropriately so bringing weechat-android to the front in the future will be optimized.

Killing processes for the sake of killing processes on Android is an anti-pattern. Again, we removed this functionality some time ago because it is an anti-pattern. So the disconnect option is still there, and I'd encourage you to use it if you want to disconnect. However, there's really no reason to "quit" an app on Android, hence why the option is gone.
